interactive/settings.py

```diff
@@ -256,3 +259,28 @@ def immutable_file_test(path, url):
 INTERNAL_IPS = [
     "127.0.0.1",
 ]
+
+# CSP
+# https://django-csp.readthedocs.io/en/latest/configuration.html
+CSP_REPORT_ONLY = DEBUG
+CSP_IMG_SRC = ["'self'", "data: w3.org/svg/2000"]
```

Comment:
Does this also need
`CSP_DEFAULT_SRC = ["'self'"]`
?

Reply:
You get +10 points on Mozilla Observatory for not including it:

> Content Security Policy (CSP) implemented with `default-src 'none'` and no `'unsafe'`

Reply:
Interestingly, the CSP being generated is 
`default-src 'self'; style-src-elem 'self'; font-src 'self'; img-src 'self' data: w3.org/svg/2000; script-src-elem 'self' https://plausible.io`

Does the default-src have a default value? i.e do you explicitly need to pass in None?